packages/backend/src/core/reports/getEBVConfigHash.ts

```diff
@@ -0,0 +1,19 @@
+import { hashJson } from '@l2beat/shared-pure'
+
+import { getTotalSupplyConfigHash } from '../totalSupply/getTotalSupplyConfigHash'
+import { TotalSupplyTokensConfig } from '../totalSupply/TotalSupplyTokensConfig'
+import { getReportConfigHash } from './getReportConfigHash'
+import { ReportProject } from './ReportProject'
+
+const EBV_LOGIC_VERSION = 0
+
+export function getEBVConfigHash(
+  project: ReportProject[],
+  tokens: TotalSupplyTokensConfig[],
+) {
+  return hashJson([
+    getReportConfigHash(project).toString(),
```

Comment:
in this case we will recalculate EBV reports when the CBV config changes, is it a desired behaviour?